dipy/workflows/viz.py

```diff
@@ -150,6 +158,36 @@ def run(self, input_files, cluster=False, cluster_thr=15.,
                     print('Peak_dirs shape')
                     print(pam.peak_dirs.shape)
 
+            if ends(".npy"):
+
+                data = np.load(fname)
+                numpy_files.append(data)
+
+                if verbose:
+                    print('numpy array length')
+                    print(len(data))
+
+        if buan:
+            bundle_colors = []
+            for i in range(len(numpy_files)):
+
+                n = len(numpy_files[i])
+                pvalues = numpy_files[i]
+                bundle = tractograms[i].streamlines
+
+                indx = assignment_map(bundle, bundle, n)
+                ind = np.array(indx)
+
+                colors = []
+                for i in range(len(ind)):
+
+                    if pvalues[ind[i]] < buan_thr:
```

Comment:
I would prefer if the highlight color was available as an option.
Also I would like if we could keep the original color of the bundle if we want.

Reply:
Done. Added original color for bundles with bundle highlight area color as an option (default red).